requirements/base.txt

```diff
@@ -9,7 +9,7 @@ nncf==2.0.0
 numpy~=1.19.5
 omegaconf==2.1.1
 pillow==8.3.2
-pytorch-lightning==1.3.6
+pytorch-lightning @ git+https://github.com/PyTorchLightning/pytorch-lightning.git
```

Comment:
Should we point it to a specific release. Like this - 
`pip install https://github.com/PyTorchLightning/pytorch-lightning/releases/download/1.5.7/pytorch-lightning-1.5.7.tar.gz`